src/library/Uno.Material/Styles/Controls/v2/ProgressRing.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:lottie_win="using:Microsoft.Toolkit.Uwp.UI.Lottie"
+					xmlns:lottie_not_win="using:Microsoft.Toolkit.Uwp.UI.Lottie"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:not_win="http://uno.ui/not_win"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+					mc:Ignorable="d not_win lottie_not_win">
+
+	<lottie_win:LottieVisualSource x:Key="M3MaterialDeterminateAnimation_Win"
+								UriSource="ms-appx:///Uno.Material/Assets/MaterialDeterminate.json" />
+	<lottie_win:LottieVisualSource x:Key="M3MaterialIndeterminateAnimation_Win"
+								UriSource="ms-appx:///Uno.Material/Assets/MaterialIndeterminate.json" />
+
+	<lottie_not_win:LottieVisualSource x:Key="M3MaterialDeterminateAnimation_Uno"
+									   UriSource="embedded://Uno.Material/Uno.Material.Assets.MaterialDeterminate.json" />
+	<lottie_not_win:LottieVisualSource x:Key="M3MaterialIndeterminateAnimation_Uno"
+									   UriSource="embedded://Uno.Material/Uno.Material.Assets.MaterialIndeterminate.json" />
+
+	<Style x:Key="M3MaterialProgressRingStyle"
+		   TargetType="muxc:ProgressRing">
+		<win:Setter Property="DeterminateSource"
+						   Value="{StaticResource M3MaterialDeterminateAnimation_Win}" />
+		<win:Setter Property="IndeterminateSource"
+						   Value="{StaticResource M3MaterialIndeterminateAnimation_Win}" />
+		<not_win:Setter Property="DeterminateSource"
+							   Value="{StaticResource M3MaterialDeterminateAnimation_Uno}" />
+		<not_win:Setter Property="IndeterminateSource"
+							   Value="{StaticResource M3MaterialIndeterminateAnimation_Uno}" />
+		<Setter Property="Foreground"
+				Value="{StaticResource PrimaryBrush}" />
+		<Setter Property="Background"
+				Value="{StaticResource PrimaryLowBrush}" />
```

Comment:
This is causing a problem in the rendering...

![image](https://user-images.githubusercontent.com/4174207/169883980-9025d0c8-f3aa-4b21-9b65-65a9ed964f6f.png)

@agneszitte-nventive